runtime/src/lib.rs

```diff
@@ -1597,6 +1600,16 @@ impl_runtime_apis! {
 		}
 	}
 
+	impl pallet_assets_ext_rpc_runtime_api::AssetsExtApi<
+		Block,
+		AccountId,
+	> for Runtime {
+		fn free_balance(asset_id: AssetId, who: AccountId, keep_alive: bool) -> String {
+			let bal = AssetsExt::free_balance(&who, &asset_id, keep_alive);
+			alloc::format!("{}", bal)
```

Comment:
I think the format is unnecessary now as the free_balance function above returns a String
```suggestion
			AssetsExt::free_balance(&who, &asset_id, keep_alive)
```

Reply:
We need this, in terms of a hex balance.. this will format it to string.